pages/linux/ncat.md

```diff
@@ -13,3 +13,7 @@
 - Write output of specified file to the specified host on the specified port:
 
 `ncat {{address}} {{port}} < {{path/to/file}}`
+
+- Connect to host/port using SSL/TLS encryption (for example for HTTPS):
```

Comment:
"Connect using ssl (for example for HTTPS):"

We have already established what {{address}} and {{port}} are for

Reply:
Hello @agnivade, thank you for your reply
resubmitted, please be so kind to close this PR - sorry for the inconvenience